config.js

```diff
@@ -529,4 +530,53 @@ var config = {
     // no configuration value should follow this line.
 };
 
+if (isCfgEnabled) {
+    // Overwrite config here...
+}
+
+function parseURLParams(
```

Comment:
The url already should parsed if the config is loaded. In the 'config.js' only need to check a parameter

Reply:
Do you mean to move this function outside this directory?